src/content/docs/usage/workspaces.mdx

```diff
@@ -118,12 +176,15 @@ Refer to **[Set the Default IDE](#set-the-default-ide)** for more information.
 
 ## Delete Workspaces
 
-You can delete one or more Workspaces interactively.
+Daytona allows you to delete one or more Workspaces, helping you manage your development environments by removing those that are no longer needed. This ensures you maintain a clutter-free and efficient Workspace.
```

Comment:
`a clutter-free and efficient Workspace` doesn't seem appropriate. I suggest removing the sentence altogether